CHANGELOG.rst

```diff
@@ -4,6 +4,18 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v7.2.0
+======
```

Comment:
This isn't needed. The content of this file is automatically generated when a release is prepared.